pkgs/by-name/fw/fw-fanctrl/package.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  python3Packages,
+  python3,
+  fw-ectool,
+  fetchFromGitHub,
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "fw-fanctrl";
+  version = "1.0.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "TamtamHero";
+    repo = "fw-fanctrl";
+    tag = "v${version}";
+    hash = "sha256-TDVULNb/oH66/UX20mC89NSx8YPe8mPwNCB9+phavP4=";
+  };
+
+  build-system = [ python3Packages.setuptools ];
+
+  dependencies = [ python3Packages.jsonschema ];
+
+  buildInputs = [ fw-ectool ];
```

Since this package is not a library and contains only a single binary, I assume you want something like this instead:

```suggestion
  makeWrapperArgs = [ "--suffix PATH : ${lib.getExe fw-ectool}" ];
```